Sources/Player/UserInterface/PictureInPictureSupportingSystemVideoView.swift

```diff
@@ -23,12 +23,29 @@ struct PictureInPictureSupportingSystemVideoView: UIViewControllerRepresentable
     let player: Player
     let gravity: AVLayerVideoGravity
 
+#if os(iOS)
     static func dismantleUIViewController(_ uiViewController: AVPlayerViewController, coordinator: ()) {
         PictureInPicture.shared.system.relinquish(for: uiViewController)
     }
+#else
+    static func dismantleUIViewController(_ uiViewController: AVPlayerViewController, coordinator: AVPlayerViewControllerSpeedCoordinator) {
+        PictureInPicture.shared.system.relinquish(for: uiViewController)
+    }
+
+    func makeCoordinator() -> AVPlayerViewControllerSpeedCoordinator {
+        .init(
+            player: player,
+            controller: PictureInPicture.shared.system.playerViewController ?? PlayerViewController()
+        )
+    }
+#endif
```

Comment:
Would avoid having two different implementations, rather make the difference with a typealias for the coordinator:

```suggestion
#if os(iOS)
    typealias Coordinator = Void
#else
    typealias Coordinator = AVPlayerViewControllerSpeedCoordinator
#endif

    static func dismantleUIViewController(_ uiViewController: AVPlayerViewController, coordinator: Coordinator) {
        PictureInPicture.shared.system.relinquish(for: uiViewController)
    }

#if os(tvOS)
    func makeCoordinator() -> AVPlayerViewControllerSpeedCoordinator {
        .init(
            player: player,
            controller: PictureInPicture.shared.system.playerViewController ?? PlayerViewController()
        )
    }
#endif
```